File: projects/pythonprojects/calculator.py
def add(a,b):
    return a+b
def sub(a,b):
    return a-b
def mul(a,b):
    return a*b
def div(a,b):
    return a/b
def mod(a,b):
    return a%b
def pow(a,b):
    return a**b
o=1
print("1)ADDITION.\n2)SUBTRACTION.\n3)MULTIPLICATION.\n4)DIVISION.\n5)MODULOUS.\n6)POWER.\n")
while o!=8008:
    print("CALCULTION : ")
    a, operation, b=input().split()
    a=int(a)
    b=int(b)
    if operation == "+":
       print("add : ",add(a,b))
    elif operation == "-":
      print(sub(a,b))
    elif operation == "*":
     print(mul(a,b))
    elif operation == "/":
      print(div(a,b))
    elif operation == "%":
     print(mod(a,b))
    elif operation == "^":
     print(pow(a,b)) 
    else:
     print(type(a),type(b))
    o=int(input("TO CONTINUE PRESS 0 TO END ENTER 8008"))

File: projects/c++projects/calculator.cpp
#include<bits/stdc++.h>
using namespace std;

float add(float a,float b){

	return a+b;
}

float sub(float a,float b){

	return a-b;
}

float mul(float a,float b){

	return a*b;
}

float div(float a,float b){

	return a/b;
}

int mod(int a,int b){

	return a%b;
}

int main(){
   float a,b;
   char c;
   int o;
   while(o!=8008){
   	cout<<"ENTER CALCULATION : ";
   	cin>>a>>c>>b;
   	switch(c){
   		case '+':
   			cout<<add(a,b);
   			break;
   		case '-':
   			cout<<sub(a,b);
   			break;
   		case '*':
   			cout<<mul(a,b);
   			break;
   		case '/':
   			cout<<div(a,b);
   			break;
   		case '%':
   			cout<<mod(int(a),int(b));
   			break;
   		case '^':
   			cout<<pow(a,b);
   			break;
	   }
	   cout<<endl<<"ENTER 8008 TO KILL PROCESS OR 0 TO CONTINUE : "<<endl;
	   cin>>o;
   }	
}
